frontend/javascripts/oxalis/view/right-border-tabs/bounding_box_tab.tsx

```diff
@@ -102,22 +103,12 @@ export default function BoundingBoxTab() {
     APIJobType.EXPORT_TIFF,
   );
 
-  const bboxInputId = (id: number) => `bounding-box-input-${id}`;
-
   // biome-ignore lint/correctness/useExhaustiveDependencies: Always try to scroll the active bounding box into view.
```

Comment:
I'd expect this is no longer relevant?

Reply:
Yeah, this was needed to be able to select the active bounding box via a getElementById and then calling scrollIntoView. But luckly I found out that antd has its own method supporting this feature which does not need an id attached to the selected table entry. The relevant effect it directly below this comment and `bboxTableRef.current.scrollTo({ key: activeBoundingBoxId });` does all the magic needed to select and highlight the bbox item in the table

Reply:
Thank you for explaining but I was referring to the `biome-ignore` comment :sweat_smile: In any case, not a big deal :)

Reply:
Oh :see_no_evil:, well according to biome the `bboxTableRef.current` dependency is unnecessary :man_shrugging:.

Sorry for that. I'll remove this in another pr :)

Reply:
Please see https://github.com/scalableminds/webknossos/pull/7886#issuecomment-2255735830